students/wachulec_maria/lesson_03_functions/the_collatz_sequence_and_input_validation.py

```diff
@@ -0,0 +1,15 @@
+def collatz(number):
+    if number % 2 == 0:
+        print(number // 2)
+        return number // 2
+    elif number % 2 == 1:
+        print(3 * number + 1)
+        return 3 * number + 1
+
+
+try:
+    n = int(input("Take me number: "))
```

Comment:
Don't mix `"` with `'` in the single file(project) for text. Be consistent. Use one convention across files.

In line 15 there is the single quote, where on line 11 there is a double quote.